front/app/routes/_index.tsx

```diff
@@ -11,41 +14,79 @@ export const meta: MetaFunction = () => {
 };
 
 export let handle = {
-  i18n: ['common'],
+  i18n: ['common', 'visitor'],
 };
 
 export default function Index() {
-  return (
-    <div className="leading-3">
-      <Heading as="h1">Welcome to Remix</Heading>
-      <ul className="flex flex-col gap-4">
-        <li>
-          <RadixLink target="_blank" href="https://remix.run/tutorials/blog" rel="noreferrer">
-            15m Quickstart Blog Tutorial
-          </RadixLink>
-        </li>
-        <li>
-          <RadixLink target="_blank" href="https://remix.run/tutorials/jokes" rel="noreferrer">
-            Deep Dive Jokes App Tutorial
-          </RadixLink>
-        </li>
-        <li>
-          <RadixLink target="_blank" href="https://remix.run/docs" rel="noreferrer">
-            Remix Docs
-          </RadixLink>
-        </li>
-      </ul>
+  const { t } = useTranslation();
 
-      <img
-        className="my-8"
-        src="/favicon.ico"
-        alt="Remix Logo"
-        style={{
-          viewTransitionName: 'logo',
-        }}
-      />
-
-      <Link to="/dashboard">To dashboard (redirect to login if not logged in)</Link>
-    </div>
+  return (
+    <>
+      <div className="relative hidden items-center overflow-hidden rem:h-[441px] lg:flex">
+        <img
+          src="/img/potatoes.jpg"
+          alt=""
+          height={1067}
+          width={1600}
+          decoding="async"
+          loading="lazy"
+          className="absolute w-full"
+        />
+      </div>
+      <Container p="3">
+        <Section className="space-y-20">
+          <Heading size="9" className="text-center capitalize">
+            symmetrical potato
+          </Heading>
+          <Text as="p">
+            <Trans i18nKey="visitor:potatoes.description"></Trans>
+          </Text>
+          <Grid gap="3">
+            <Heading as="h2" size="8" className="text-center first-letter:uppercase">
+              {t('visitor:potatoes')}
```

Comment:
```suggestion
              {t('potatoes', {
                 ns: 'visitor'
               })}
```

Pareil pour les autres